src/test/java/com/google/sps/AttendeeTest.java

```diff
@@ -0,0 +1,54 @@
+import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Attendee;
+import com.google.sps.data.AttendeeInterface;
+
+@RunWith(JUnit4.class)
+
+public class AttendeeTest {
```

Comment:
Weird spacing. Should be:

@RunWith()
public class AttendeeTest

Reply:
Also this is a class so please write a comment explaining what the class is and what it does just like your other classes. Put the comment on top of the @ declaration. 

[the comment goes here]
@RunWith()
public class AttendeeTest

Reply:
Is "Class that test the methods in the Attendee class" appropriate or should I include interface

Reply:
thats fine